integration-tests/entry-point.sh

```diff
@@ -60,6 +57,24 @@ chmod 640 ${CONFIG_PATH}
 # start ssh
 service ssh start || true
 
+# Ensure lockfile exists
+mkdir -p /run/lock > dev/null
+touch /run/lock/devicecl.lock
+
+# Spoof lockfile
+echo "${THING_NAME}" > /run/lock/devicecl.lock
```

Comment:
I'm confused about what exactly is supposed to happen here.

Reply:
We now check for thing name as well as pid since we may want to support the use case of running multiple DC instances with different thing names